automated-comments/go/hamming/comparing_bytes_disguised_as_runes.md

```diff
@@ -0,0 +1,4 @@
+Converting the `bytes` to `runes` to compare disguises the problem.
+It works perfectly in the context of this exercise as there are only ascii letters in DNA strands.
```

Comment:
I think we would usually write 'ASCII', not 'ascii'—it's an acronym.

Reply:
```suggestion
It works perfectly in the context of this exercise as there are only ASCII letters in DNA strands.
```